src/openapi/spec/api-request-schema.ts

```diff
@@ -0,0 +1,127 @@
+import { Operator } from 'unleash-client/lib/strategy/strategy';
+import { createSchemaObject, CreateSchemaType } from '../openapi-types';
+
+const schema = {
+    type: 'object',
+    required: ['features', 'version'],
+    properties: {
+        version: {
+            type: 'integer',
+            enum: [2],
+        },
+        features: {
+            type: 'array',
+            items: {
+                type: 'object',
+                required: ['name'],
+                additionalProperties: false,
+                properties: {
+                    name: {
+                        type: 'string',
+                    },
+                    description: {
+                        type: 'string',
+                    },
+                    enabled: {
+                        type: 'boolean',
+                    },
+                    stale: {
+                        type: 'boolean',
+                    },
+                    impressionData: {
+                        type: 'boolean',
+                    },
+                    strategies: {
+                        type: 'array',
+                        items: {
+                            type: 'object',
```

Comment:
Maybe we could move this to a file/ref for readability, something like:

https://github.com/Unleash/unleash/blob/main/src/lib/services/strategy-schema.ts

Reply:
Yeah, about that. I thought about it, but I think the readability there is subjective. I'd much rather keep it in the same file so I don't have to try and find the correct file for the ref to see what's going on 💁🏼 But again, I'm happy to conform to the pattern if we think it's better.

Or a compromise: separate schema in the same file?

Reply:
Definitely, either way works for me. 

On the backend I think they should be separate since we probably want to reuse the strategies/variants schemas in multiple other schemas, while also having them show up as the same thing in the spec UI. But the proxy can be simplified.

Reply:
Yup, that was my thinking too. We only use these types in one place, so splitting them up only makes things more confusing (at least to me 🙋🏼 ), so I'll keep this as it is for now.